README.md

```diff
@@ -54,18 +54,21 @@ poetry install
 
 ## Configuring the application
 
-This program uses environment variables to handle settings. Copy `.env.example` into a new `.env` file and edit it as needed.
+This program uses environment variables to handle settings. 
+Copy `.env.example` into a new `.env` file and edit it as needed.
 
 ```bash
 cp .env.example .env
 ```
 
-By default, `.env` is configured: 
-- To use [intfloat/e5-large-v2](https://huggingface.co/intfloat/e5-large-v2) as an embedding model
-- To connect to a local instance of [Ollama](https://ollama.ai) for inference
-- To use a basic, fairly neutral retrieval prompt 
+See details for individual settings in [.env.example](.env.example).
 
-Edit this file as needed to adjust settings, replace the embedding model, retrieval prompt, or to connect WARC-GPT to [Open AI](https://platform.openai.com/docs/introduction), [Anthropic](https://docs.anthropic.com/claude/reference/getting-started-with-the-api), [Cohere](https://docs.cohere.com/docs) or [Perplexity AI](https://docs.perplexity.ai/).
+**A few notes:**
+- WARC-GPT can interact with both the [OpenAI API](https://platform.openai.com/docs/introduction) and [Ollama](https://ollama.ai) for local inference. 
+  - Both can be used at the same time, but at least one is needed. 
+  - By default, the program will try to communicate with Ollama's API at `http://localhost:11434`.
+  - It is also possible to use OpenAI's client to interact with compatible providers, such as [HuggingFace's Message API](https://huggingface.co/blog/tgi-messages-api) or [vLLM](https://docs.vllm.ai/en/latest/getting_started/quickstart.html#using-openai-completions-api-with-vllm). To do so, set values for both `OPENAI_BASE_URL` and `OPENAI_COMPATIBLE_MODEL` environment variables. 
```

Comment:
I left that aside because what to do with these two variables depend on the target platform. For example, one can use the OpenAI client with Ollama, and is supposed to use `"ollama"` as an API key.